crates/llms/src/chat/mod.rs

```diff
@@ -93,10 +95,58 @@ pub enum Error {
 
     #[snafu(display("The specified model, '{from}', does not support executing the task '{task}'.\nSelect a different model or task, and try again."))]
     UnsupportedTaskForModel { from: String, task: String },
+
+    #[snafu(display("Failed to find tensors for the model.\nExpected tensors with a file extension of: {extensions}.\nVerify the model is correctly configured, and try again."))]
```

Comment:
They're more commonly called weights rather than tensors. That's also the file type we use in the spicepod :https://spiceai.org/docs/reference/spicepod/models#files